blog/2020-12-21-openhab-3-0-release.md

```diff
@@ -0,0 +1,89 @@
+
+The sweet spot for openHAB 3 is in its balance: The underlying framework has been modernized without any negative impact, a lot of complexity has been thrown overboard without dropping features that our users rely on and many things have been streamlined, all of which has enabled our completely new main user interface.
+
+## Architecture Evolution
+
+Version 1 of openHAB consisted of a clean and simple architecture that worked well up to the moment that more "main stream" users joined the community and complemented the existing developer-centric crowd.
+
+Version 2 of openHAB had to compromise between the "old" purely textual openHAB 1 world and the new UI-enabling features that were introduced with Eclipse SmartHome. The way one had to work with bindings and rules very much differed from each other and these two world existed more or less side-by-side, glued together through the 1.x compatibility layer.
+
+With openHAB 3 a huge part of its simplification is due to the removal of the 1.x compatibility layer. All bindings now work exactly the same way and the user decides on how they wish to configure their system, either textually or through a UI. To make sure that nobody is left behind by this move, [many 1.x bindings have been reimplemented](https://github.com/openhab/openhab-addons/issues/6179#migrated) for openHAB 3. For the rest, it is possible to keep an old 2.x instance running while integrating it into a new 3.0 setup - see the [openHAB outposts](#outposts) below.
+
+Furthermore, the two separate rule engine implementations have been merged into one, so that there is now a single concept for automation rules. As a consequence, textual rules written in openHAB's own Rule DSL are also available within the user interface. 
+
+Besides the visible changes, many refactorings have been done to the codebase in order to keep it modern and maintainable. Moving from Java 8 to Java 11 brings the language features on a current level and ensures that openHAB continues to run on JVMs that receive security patches. Additionally, many included libraries have been upgraded, deprecated methods were removed, null annotations introduced, newer OSGi features used and so on. Overall, the codebase is a future-proof and solid foundation to further build on for the years to come.
+
+<p align="center"><video autoplay loop src="/uploads/evolution.mp4"></video></p>
+
+### Outposts
```

Comment:
I must admit I find that term a little confusing in this context... but I'm not an English native speaker :)

Reply:
I was inspired by https://aws.amazon.com/de/outposts/ ;-)

Reply:
Hmm, i thought maybe the remote openHAB binding used this term.  I agree it may be a little confusing, maybe call them "secondary" instances ?  Non primary ?